content/liveobjects/storage.textile

```diff
@@ -0,0 +1,62 @@
+---
+title: Object storage
+meta_description: "Learn about LiveObjects object storage."
+product: liveobjects
+languages:
+  - javascript
+---
+
+<aside data-type='experimental'>
+<p>LiveObjects is currently Experimental. Its features are still in development and subject to rapid change.</p>
+</aside>
+
+h2(#default-persistence). Default object storage - 90 days
```

Comment:
I wonder if we should define a term here, eg the "object retention period", and then use that term in the remainder of the doc instead of repeating 90 days (which won't be correct for all accounts).

So here the text says something like

> Ably durably stores all objects on a channel for a retention period that is configured to 90 days by default. If the collective object state for a channel is not updated within the retention period, it will automatically expire. ...

Reply:
Yeah I think that's fair, but currently it's not configurable so I've opted to keep it simple. When we implement configurable retention & limits by package type, we can update the docs

Reply:
Fixed in: [5034056](https://github.com/ably/docs/pull/2591/commits/5034056be796fbc27dd4f1d21b51b3d763ef90cb)